internal/generator/vector/normalize.go

```diff
@@ -57,10 +57,8 @@ func NormalizeContainerLogs(inLabel, outLabel string) []generator.Element {
 			Inputs:      helpers.MakeInputs(inLabel),
 			VRL: strings.Join(helpers.TrimSpaces([]string{
 				FixLogLevel,
-				RemoveFile,
```

Comment:
please do not change the data model. a change here affects all outputs not just cloudwatch